Framework/DataHandling/test/LoadTest.h

```diff
@@ -233,6 +233,30 @@ class LoadTest : public CxxTest::TestSuite {
                boost::algorithm::ends_with(second, "CSP79590.raw"));
   }
 
+  /*
+   * This test loads and sums 2 IN4 runs from ILL
+   * without instrument prefix in the file names.
+   */
+  void test_ILLLoadMultipleFilesNoPrefix() {
+    Load loader;
+    loader.initialize();
+    loader.setPropertyValue("Filename", "ILL/IN4/IN4084446+084447.nxs");
```

Comment:
Is this how we expect users would use this? Would is not be better to require that the directory containing the files is in the `datasearch.directories` path and then simply set `loader.setPropertyValue("Filename", "084446+084447.nxs");`